registration/models.py

```diff
@@ -115,6 +116,12 @@ class Field(models.Model):
     question = models.CharField(max_length=512)
     question_type = models.CharField(max_length=15, choices=QUESTION_CHOICES)
     is_default = models.BooleanField()
+    options = ArrayField(
+        models.CharField(max_length=64, blank=False),
+        validators=[validate_mc_options],
+        null=True,
+        blank=True,
```

Comment:
what do you think about making this an empty list by default, and disallowing null values? it may make the logic cleaner for the form fields settings later (ie. changing a question type between text & mc)

Reply:
that's a really good point. I think yeah, it'll be simpler to work with in the future.